apps/condo/domains/ticket/access/TicketReportService.js

```diff
@@ -0,0 +1,13 @@
+async function canReadTicketReportWidgetData ({ authentication: { item: user } }) {
+    if (!user) return false
+
+    if (user.isAdmin) {
+        return {}
+    }
+
+    return { organization: { employees_some: { user: { id: user.id }, isBlocked: false } } }
```

Comment:
I think we should add deletedAt: null here, because otherwise users deleted from organization might be able to get information